pkg/component/controller/etcd.go

```diff
@@ -107,6 +107,8 @@ func (e *Etcd) syncEtcdConfig(ctx context.Context, etcdRequest v1beta1.EtcdReque
 			etcdResponse, err = e.JoinClient.JoinEtcd(ctx, etcdRequest)
 			return err
 		},
+		retry.Delay(1*time.Second),
```

Comment:
Can you explain why the delay was increased in the commit message, or in a comment? If I'm not mistaken this will now block for ~ 17 minutes, whereas it was blocking only around 100 secs before?

Reply:
I did the calculation again, as I forgot to take into account the max delay. I think it's 5 minutes overall, not 15 ... :angel:

Reply:
I increased the number of attempts to 20. So I think it's 127 seconds plus like 12 minutes

Reply:
Okay. Will it be likely that joining will succeed after 15 minutes, when it didn't after 5 minutes? I'm just thinking if it makes sense to wait for that long, as surrounding tooling may have its own, shorter timeouts. How long does k0sctl wait until it aborts the join process? /cc @kke

Reply:
Ping @kke @twz123 . I'm waiting on a response to the above ^

Reply:
So what is the actual timeout now with this implementation? I don't think it makes sense to have it more than say 5mins, if the joining won't work in the first 5 mins, what would make it work after that?

Reply:
@jnummelin ive decreased it to ~ 4 minutes